src/DependencyInjection/SonataAdminExtension.php

```diff
@@ -68,14 +68,14 @@ public function load(array $configs, ContainerBuilder $container): void
         $loader->load('twig.xml');
         $loader->load('validator.xml');
 
-        if (isset($bundles['MakerBundle'])) {
-            $loader->load('makers.xml');
-        }
-
         if (isset($bundles['SonataExporterBundle'])) {
             $loader->load('exporter.xml');
         }
 
+        if (isset($bundles['MakerBundle'])) {
```

Comment:
dont change this order

Reply:
Asked here https://github.com/sonata-project/SonataAdminBundle/pull/6093#discussion_r427772054

Reply:
As it is not related to make this work for sf 5, it is better to revert it. It can be done on another PR